great-idea/services.html

```diff
@@ -1,87 +1,148 @@
+
+    </section>
+
+
+    
+    
+    
+    <section class="UX">
+    
+    <h1>UX / UI</h1>
+    
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p>
+    
+    <button>Learn More</button>
+  </section>
+    
+    <section class="DM">
+    <h1>Digital Marketing</h1>
+    
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p>
+    
+    <button>Learn More</button>
+  </section>
+    
+    <section class="WD">
+    <h1>Web Development</h1>
+    
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p>
+    
+    <button>Learn More</button>
+  </section>
+    
+    <section class="Apple">
+    <h1>iOS Development</h1>
+    
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p>
+    
+    <button>Learn More</button>
+    
+  </section>
+    
+  <section class="Droid">
+
+
+    <h1>Android Development</h1>
+    
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p>
+    
+    <button>Learn More</button>
+  </section>
+  
+
+
+    <section class="facts">
+    <h1>Some Facts About Our Services</h1>
+    
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.
+      </p>
+    
+   <ul>
+     <li>Awesome thing</li>
+   <li>Amazing thing</li> 
+    <li>Cool thing</li>
+    <li>Great thing</li>
+  </ul> 
+    
+  </section>
+  <section class="pic">
+  <img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
+</section>
+
+
+</section>
+</div>
```

Comment:
you could leave this out if you like - the footer can be it's own container in this instance.